coordinator/core/clientapi_test.go

```diff
@@ -73,9 +75,30 @@ func TestSetManifest(t *testing.T) {
 		break
 	}
 	modRawManifest, err := json.Marshal(manifest)
-	assert.NoError(err)
+	require.NoError(err)
 	_, err = c.SetManifest(context.TODO(), modRawManifest)
 	assert.Equal("manifest does not contain marble package foo", err.Error())
+
+	// Try setting manifest with UniqueID + other value set, no debug mode (this should fail)
```

Comment:
In addition to what you have implemented and test here, we also want to enforce that if UniqueID is not set, the other 3 values must all be set. So setting, e.g., only SignerID and ProductID is invalid.
When you add this functionality and more tests, it is probably a good idea to move these test into its own Test... func.